src/lib/features/playback-controls/ui/full-controls.svelte

```diff
@@ -1,24 +1,28 @@
 <script lang="ts">
-  import IconReplay10 from '~icons/ic/round-replay-10';
-  import IconForward30 from '~icons/ic/round-forward-30';
   import {
     seek,
-    move,
     currentTime,
     duration,
-    paused,
     buffered,
     pause,
     play,
+    playbackRate as speed,
   } from '$lib/entities/audio';
-  import { IconButton } from '$lib/shared/ui/';
-  import PlaybackButton from './playback-button.svelte';
-  import { toggleGlobalPlayback } from '../model/toggle-global-playback';
   import ScrubbingBar from './scrubbing-bar.svelte';
   import clsx from 'clsx';
+  import PlaybackControlButtons from './playback-control-buttons.svelte';
+  import LikeButton from '$lib/features/like-episode/ui/like-button.svelte';
+  import type { Episode } from '$lib/shared/api/types';
 
   let _class = '';
   export { _class as class };
+  export let prominent = false;
+  export let episode: Episode | undefined;
+
+  function nextSpeed() {
+    $speed = ($speed + 0.5) % 2.5;
```

Comment:
issue (non-blocking): I understand that this probably works, but it updates the store twice and just generally looks very complex

suggestion (non-blocking): maybe we could have an array of options and use that for cycling?